indico/htdocs/js/indico/modules/categories/categories.js

```diff
@@ -18,6 +18,17 @@
 (function(global) {
     'use strict';
 
+    $(document).ready(function() {
+        setupCategoryMoveButton();
```

Comment:
I don't like this. I'd rather make this function global (`global.setupCategoryMoveButton = function setupCategoryMoveButton() {...};`) and call it from a script tag on the pages where we need it.